azerothcore/azerothcore-wotlk
# (DB/Translation) Howling Fjord missions translated into Spanish

## Changes Proposed:
-  I was reviewing most of the missions in the area, noticing that many of their texts were in English, even though I was using a Spanish game client.
-  In the file, you can find most of the horde, alliance and neutral missions of the map. It is not a sniff, but for the moment, it provides a solution to the missions that are untranslated. Later, they can be improved if necessary.

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Each of the missions was tested separately within the game. What I did, was to create a level 80 character, with enough equipment to be able to do them, and I set out to do them. Some of them I completed with the `quest complete` command, but the vast majority of them I did as a player.

## How to Test the Changes:


1. You should have done the missions currently in the emulator.
2. Then, apply the SQL and notice that many of your texts will be in Spanish. The objectives may not be in Spanish, in that case, you will have to close the game and delete the cache folder to see the results.
3. Personally I took the trouble to try them one by one, although if you want to give it a try, all the better. In fact, I have all the missions in separate files. I just wasn't going to upload them that way, I prefer to upload them in a single file.

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
